ddtrace/contrib/falcon/middleware.py

```diff
@@ -21,6 +23,8 @@ def __init__(self, tracer, service="falcon", distributed_tracing=False):
             service=service,
         )
 
+        self.traced_headers = config.http.get_integration_traced_headers('falcon')
```

Comment:
Would it be good to have a few tests configuring and asserting that the headers are set in falcon?

Reply:
👍 tests will be added